Content.Server/GameObjects/Components/Observer/GhostComponent.cs

```diff
@@ -68,9 +70,23 @@ public override void HandleNetworkMessage(ComponentMessage message, INetChannel
                         Owner.Delete();
                     }
                     break;
+                case ReturnToCloneComponentMessage reenter:
+                    Owner.TryGetComponent(out VisitingMindComponent mind);
```

Comment:
This needs to be wrapped in an if or `GhostReturnMessage.Mind` needs to be made nullable.